### DISCUSSION TITLE
Q4S2 (Mid-Nov to Dec) Roadmap Planning

### DISCUSSION THREAD

**Discussion by BenJam** [opencollective/opencollective#4951] [Ideas]
![Slice 1](https://user-images.githubusercontent.com/158833/141981023-5deee07f-5b90-4459-8389-d2b7b665724c.png)

_It's the most wonderful ~~time~~  sprint of the year..._ that's right it's the last sprint of the year, a time for looking back at the year past and looking forward to the year to come. 

[Last sprint](https://github.com/opencollective/opencollective/issues/4783) we expanded the scope of our virtual cards feature, adding issuing through Stripe. This addresses a significant policy problem with our existing provider, which has been causing considerable pain for Open Collective Foundation. We also invested a significant amount of time in supporting hosts with a new dashboard and improved transaction export features. Finally we made some small but impactful changes to protect user privacy. On the design side we started looking at how we talk about ourselves and our work, improve our onboarding process for different types of users and improving the ability for Collectives to tell their stories too. 

## Our priorities

> Our priorities do not change, but our framing does. We continue to focus on our core strengths but as a product team, we shall consolidate on our existing success.

We should continue to focus on fundamentals like the ledger, completing unfinished projects and, addressing issues that cause ourselves and our user's significant pains and limit our growth. We should try not to focus on the needs of smaller hosts, self-hosted collectives, and significant, new feature development.

## Non-negotiables and limiting proposals to top priorities only

We anticiapte that we will continue to dedicate a significant amount of the sprint to continue work on:

- Virtual card issuing with stripe #4807 
- Host reports 
- User onboarding #4829 
- Homepage design #4689 

Of course we anticiapte that we will be working at a significantly reduced capcity in the the final two weeks of this spring, we also have a week (6th Dec) dedicated to our second asynchronous retreat. 

**As such we encourage you to only propose your top 1-2 priorities for the sprint**

## How do I contribute?

This is the _fifth_ time we've used GitHub Discussions to collect ideas and vote on priorities, so members of the core team should be used to the mechanics (described below). For our hosts who wish to contribute to the dicussion:  

#### Proposing issues
Create a top-level comment on this discussion, link to any issues that already exist in this repo (a quick search is good enough). Bear in mind issues that were discussed in previous sprints that we may have [added to the roadmap](https://github.com/opencollective/opencollective/projects/54). If you can't find an existing issue create one (if it's a dupe we'll merge them later) using one of the [user need](https://github.com/opencollective/opencollective/issues/new?assignees=&labels=&template=user_need.md&title=) (if you aren't proposing a solution) or [feature spec](https://github.com/opencollective/opencollective/issues/new?assignees=&labels=feature&template=feature-spec.md&title=) (if you have something in mind) templates.  

#### Triaging issues
The following triage heuristic which should be applied to issues proposed for a sprint. There's [an explainer on Loom](https://www.loom.com/share/369ab467fbc64dec848085d38ff57ca0) if you'd like to know more. 

<details>
<summary>Triage Template</summary>
Problem, user need or idea:
Proposed solution:

P1 high frequency high impact
P2 low frequency, high impact
P3 high frequency, low impact
P4 low frequency, low impact

Examples of high impact - a problem affects users during an essential process (expenses and payments > onboarding and registration > contributing) with no workaround. High frequency - >10% of users affected (measured as a proportion of total potential users for this case). 
</details>

#### Scoping the problem
If you feel that the idea for a project or a specific issue isn't well-scoped then feel free to loop in others to help scope it down. This might involved validating the problem with some quantitative research, asking someone whether they've encountered anyone else with a similar problem or reaching out to users specifically to check an idea or validate an assumption. There is no need to scope the solution at this stage, we are looking for the most impactful problems to solve that are in line with our mission, our values, and our current focus. 

#### Discussion
Feel free to lend thoughts, opinions, data and ask questions in threads from each of the top-level comments (which should contain the proposed project).

#### Voting
Feel free to +1 any project ideas by hitting the ⬆️  button alongside any top-level comment. Try to limit yourself to three votes for the sprint to keep things focussed.

#### Synchronous discussion
We'll arrange a synchronous discussion for everyone **Wednesday 17th November** to talk through the top priorities for the sprint.

#### Kickoffs and further discussion
Will contnue for the remainder of the week.

**Comment by Betree** [comment]
# Unable to pay 100% of collective balance due to fees https://github.com/opencollective/opencollective/issues/1073 ✅

This one should be simple enough to solve and has been annoying support for years. We pretty much agreed already that we wanted to prioritize it asap.

**Comment by Betree** [comment]
# HelloWorks: send tax form request email ourselves https://github.com/opencollective/opencollective/issues/3779 ✅

This one has a `medium` complexity. I think it's important because:
- It will make the whole tax form experience clearer / easier for users
- It will resolve a confusion with branding. A user recently reported that this confusion could be abused for phishing attacks.

**Comment by Betree** [comment]
# Collective background image: responsive cropper https://github.com/opencollective/opencollective/issues/4140 ✅

`medium` complexity, shouldn't take more than 1-2 days.

I think this one is important because we keep seeing collectives with poorly cropped images, ending up with blank spaces that look really non-professional on the collective page. It's not the admins' fault: in most cases, people resize the images on a smaller screen and it looks good on their machine, the issue then appears when you look at it from a bigger screen.

The longer we wait to tackle this, the more cases of badly cropped images we will see.

**Comment by SudharakaP** [comment]
# Donations on Behalf of Someone (Scoping) ❌

Maybe we can scope out adding support for donations on behalf of someone. 

Reference Discussion: https://github.com/opencollective/opencollective/discussions/4851

**Comment by znarf** [comment]
### Expenses across Hosts ✅

This is a requirement for Google Season of Docs and should have a relatively low complexity if based on simple Bank Transfers between Hosts.

This can also serve as a basis for an Expense based Contribution across Hosts.

https://github.com/opencollective/opencollective/issues/4497 is also somehow related to this.

**Comment by znarf** [comment]
### Financial Contributions across Hosts (scoping) ✅

Given latest feedback, we could have another look at scoping and prepare it for next year.

https://github.com/opencollective/opencollective/issues/3190

**Comment by znarf** [comment]
### Virtual Cards with Stripe ✅

Complete the project and go the extra mile polishing the feature.

**Reply by alanna** [reply]
Will the implementation with Stripe enable non-US hosts to have virtual cards? I think we should do it either way, but if it enables more hosts it's even more of an opportunity.

**Reply by znarf** [reply]
@alanna There is no guarantee, as Stripe is not enabling the feature for all non-US based accounts. If the Hosts succeed activating the feature (by applying on Stripe), then this will be available for them.

**Comment by znarf** [comment]
### New Payment Methods using Stripe (scoping) ✅

It's still a pity that we're not supporting faster new Payment Methods given what Stripe is potentially supporting.

There has been some new developments at Stripe and would be good having another look at what we could do.

See https://stripe.com/en-de/payments/elements

**Comment by znarf** [comment]
### Expenses X Projects/Events ✅

We have some users complaining that it's not possible to have Projects as Payee. Would be great to look at this and more generally at the payee situation (clarify which profiles are allowed to be a payee, Invite vs New/Edit expense). There is a lot of uncertainty and confusion going on there.

https://github.com/opencollective/opencollective/issues/4920

**Reply by alanna** [reply]
I would love us to work toward a time when the payee can be any Collective, Project, or Event across any host. This will remove friction and take advantage of OC as a flexible internal economy. We need to make it easier to move money between these Collective types and this will help.

**Comment by alanna** [comment]
@BenJam can you confirm whether the items with ✅ on the [previous OCF priorities](https://github.com/opencollective/opencollective/discussions/4746#discussioncomment-1417479) have indeed been done or are in the plan still to get done? If any have not been completed we should prioritize finishing them.

**Reply by alanna** [reply]
I will bring forward some items that didn't make the cut last time from the OCF priority list for reconsideration below.

**Reply by alanna** [reply]
I discussed with @BenJam and we can confirm all the info required from those previous priorities has been brought into this current discussion.

**Comment by alanna** [comment]
## [Refund added funds transactions](https://github.com/opencollective/opencollective/issues/4369) ✅

This seems like a relatively small thing that would save support time. As this payment method is the easiest for us to refund, I don't understand why it's not possible.

**Comment by alanna** [comment]
## [Host can automatically tweet when a new Collective is approved](https://github.com/opencollective/opencollective/issues/4765) ❌

We already have a Twitter integration (with other triggers), and we already have the twitter for many Collectives, so this seems like it wouldn't be too hard. OCF is pushing social media a lot more and this would help.

**Comment by alanna** [comment]
## [Host can set custom manual payment methods](https://github.com/opencollective/opencollective/issues/4024) ✅

This would massively benefit OCF (because we have to process a lot of checks), and I think it would be beneficial to many other hosts, especially those in places with more common local payment methods we don't integrate with. This doesn't require any technical integration with third party tools, just repurposing the bank transfer instructions box and email notifications for more adaptable purposes.

**Comment by alanna** [comment]
## [Search based on tags like categories, locations](https://github.com/opencollective/opencollective/issues/4759) ❌

We are in the middle of a big project to add tags to all OCF Collectives (led by @kaylarep) and are working with @Betree to do this manually in the DB. Once this is done, to really take advantage of it, we need to be able to search based on tags. This will also be very beneficial to other hosts who want to better understand their hosted groups, and people from topic-based communities (e.g. XR, python, london, any number of topics).

**Reply by BenJam** [reply]
We have a design for this, it's a big lift though.

**Reply by kaylarep** [reply]
any idea what it would take to get this? (if not this sprint, would it be the next one? or is it a bigger project than even that?)  I understand its not so urgent compared to other issues, but this could really improve a user's experience. I imagine most of our users are pretty focused on their initiative, complete their work on the platform, and leave.

I'm thinking more of donors who contribute to one group and get curious about what else they can do on OC and might feel pretty confused/limited as it is right now. Or random people who end up on OC and want to look around, but don't really know where to go (and then can't really see the whole spectrum of amazing work we host).

I think it might be better to hold off on manually tagging groups until people can search using them. (that way, we'd have a more up-to-date amount of tagged groups)

**Comment by alanna** [comment]
## [Edit pending order](https://github.com/opencollective/opencollective/issues/4069) ✅

This is another one that would save time for support (and engineering) if we didn't need to manually get the DB edited every time.

**Comment by Memo-Es** [comment]
## Implement https://github.com/opencollective/opencollective/issues/3081 and https://github.com/opencollective/opencollective/issues/3328 ✅
 
* Customize number of expense approvals
* Two to sign for payouts

**Reply by alanna** [reply]
Reminder: this was OpenMined's top request for platform dev previously

**Comment by Memo-Es** [comment]
## Homepage (Continuation) ✅

* Produce the high-quality mockups for the new homepage https://github.com/opencollective/opencollective/issues/4689
* Better sharing https://github.com/opencollective/opencollective/issues/4762

**Comment by Memo-Es** [comment]
## Design work on expenses and transactions ✅

* Production ready files of the new additions of the transaction list
* Merge new transactions UI in the DS
* Expense flow UX follow-up
* https://github.com/opencollective/opencollective/issues/1073

**Comment by BenJam** [comment]
## [Ethical policy](https://github.com/opencollective/opencollective/issues/4928) ✅

- publish an ethical policy that clarifies our position onthe things that we want to see on the platform

**Comment by BenJam** [comment]
## Improve and standardise terminology ✅

- make the terminology easier to understnad and use it consistently across the platform https://github.com/opencollective/opencollective/issues/4929

**Comment by kaylarep** [comment]
## [Edits to OCF's application flow](https://github.com/opencollective/opencollective/issues/4943)

- Dead link is the top priority please!

**Reply by BenJam** [reply]
I think Suds has already hit this one 🙌

**Reply by SudharakaP** [reply]
Yes this is almost done. We can merge this next week I think. 👍🏽
